HerPublicWebsite/Models/Questionnaire/EligibleViewModel.cs

```diff
@@ -25,4 +25,5 @@ public class EligibleViewModel : QuestionFlowViewModel
 
     public bool IsEmailAddressRequired => CanContactByEmail is YesOrNo.Yes;
     public bool IsPhoneRequired => CanContactByPhone is YesOrNo.Yes;
+    public bool DontValidatePhoneNumber =>!IsPhoneRequired;
```

Comment:
Couple of minor things - missing a space, and the name is inconsistent with the `DoNotValidate...` attribute